frontend/src/components/Search/SearchForm/SearchForm.css.ts

```diff
@@ -184,7 +184,23 @@ export const comboboxGroup = style([
 		},
 	},
 ])
-
 export const toggle = style({
 	paddingLeft: '8px',
 })
+
+export const showAllResults = style({
+	cursor: 'pointer',
+	display: 'inline-block',
+	alignItems: 'center',
+	padding: '0 10px',
```

Comment:
nice, looks like a significant improvement

before

<img width="343" alt="Screenshot 2024-07-11 at 17 31 03" src="https://github.com/highlight/highlight/assets/1351531/16588ea0-c5c0-4b09-b286-cebc1d1d5f96">

after

<img width="325" alt="Screenshot 2024-07-11 at 17 29 56" src="https://github.com/highlight/highlight/assets/1351531/61783def-1e47-4375-86eb-afdcd03d0e14">

@rajeshj11 is it possible to clean up the spacing between `show all results` and the filter? looks like in the vertical dimension the rows are too close together.

Reply:
Ack